src/style.css

```diff
@@ -1 +1,68 @@
-/* Styles go here */
+* {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+  line-height: 60px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  cursor: pointer;
+}
+
+.nav-ul {
```

Comment:
```suggestion
.nav__ul {
```

Reply:
change it everywhere.
and
![image](https://user-images.githubusercontent.com/98764641/197799856-79128827-f508-4c39-aa0d-987158ef8c58.png)